src/PrestaShopBundle/Controller/Admin/ProductImageController.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * 2007-2015 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2015 PrestaShop SA
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+namespace PrestaShopBundle\Controller\Admin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Validator\Constraints as Assert;
+use PrestaShopBundle\Form\Admin\Type\TranslateType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+
+/**
+ * Admin controller for product image
+ */
+class ProductImageController extends FrameworkBundleAdminController
+{
+    /**
+     * Manage upload for product image
+     *
+     * @param int $idProduct
+     * @param Request $request
+     *
+     * @return string
+     */
+    public function uploadImageAction($idProduct, Request $request)
+    {
+        $response = new JsonResponse();
+        $adminProductWrapper = $this->container->get('prestashop.adapter.admin.wrapper.product');
+        $translator = $this->container->get('prestashop.adapter.translator');
+        $return_data = [];
+
+        if ($idProduct == 0 || !$request->isXmlHttpRequest()) {
+            return $response;
+        }
+
+        $form = $this->createFormBuilder(null, array('csrf_protection' => false))
+            ->add('file', 'file', array(
+                'error_bubbling' => true,
+                'constraints' => [
```

Comment:
This will force any user that want to customize theses messages and file limit to hardcode this class.

Maybe can we think about a configuration file to allow change any of theses values: what do you think ?